documentation/modules/deploying/con-deploy-kafka-connect-config-connector-api.adoc

```diff
@@ -0,0 +1,60 @@
+
+[role="_abstract"]
+Strimzi provides two APIs for creating and managing connectors:
+
+* `KafkaConnector` resources (referred to as KafkaConnectors)
+* The Kafka Connect REST API
+
+Using the APIs, you can:
+
+* Check the status of a connector instance
+* Reconfigure a running connector
+* Increase or decrease the number of connector tasks for a connector instance
+* Restart connectors
+* Restart connector tasks, including failed tasks
+* Pause a connector instance
+* Resume a previously paused connector instance
+* Delete a connector instance
+
+.KafkaConnector custom resources
+
+KafkaConnectors allow you to create and manage connector instances for Kafka Connect in a Kubernetes-native way, so an HTTP client such as cURL is not required.
+Like other Kafka resources, you declare a connector’s desired state in a `KafkaConnector` YAML file that is deployed to your Kubernetes cluster to create the connector instance.
+`KafkaConnector` resources must be deployed to the same namespace as the Kafka Connect cluster they link to.
+
+You manage a running connector instance by updating its corresponding `KafkaConnector` resource, and then applying the updates.
+You remove a connector by deleting its corresponding `KafkaConnector`.
+
+To ensure compatibility with earlier versions of Strimzi, KafkaConnectors are disabled by default.
+To enable KafkaConnectors for a Kafka Connect cluster, you annotate the `KafkaConnect` resource with the `strimzi.io/use-connector-resources` annotation.
+For instructions, see link:{BookURLUsing}#proc-kafka-connect-config-str[Configuring Kafka Connect^].
+
+When KafkaConnectors are enabled, the Cluster Operator begins to watch for them.
+It updates the configurations of running connector instances to match the configurations defined in their KafkaConnectors.
+
+Strimzi provides xref:deploy-examples-{context}[example configuration files], including the following example `KafkaConnector` file:
+
+* `examples/connect/source-connector.yaml`.
+
+You can use this example to create and manage a `FileStreamSourceConnector` and a `FileStreamSinkConnector`, as described in xref:proc-deploying-kafkaconnector-{context}[Deploying the example `KafkaConnector` resources].
+
+NOTE: You can xref:proc-manual-restart-connector-str[restart a connector] or xref:proc-manual-restart-connector-task-str[restart a connector task] by annotating a `KafkaConnector` resource.
+
+.Switching from the Kafka Connect API to using KafkaConnectors
+
+You can switch from using the Kafka Connect API to using KafkaConnectors to manage your connectors.
+To make the switch, do the following in the the order shown:
+
+. Deploy `KafkaConnector` resources with the configuration to create your connector instances.
+. Enable KafkaConnectors in your Kafka Connect configuration using the `strimzi.io/use-connector-resources` annotation.
++
+If you enable KafkaConnectors before creating the resources, you will delete all your connectors.
```

Comment:
Should this be a `WARNING:` (or whatever the syntax is)?

Reply:
Changed to a _WARNING_